src/components/resetPassword/ResetPasswordPage.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { toast } from 'react-toastify';
+import PropTypes from 'prop-types';
+import ResetPasswordForm from './ResetPasswordForm';
+import { resetPassword } from '../../actions/userActions';
+
+export class ResetPasswordPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password1: '',
+      password2: '',
+    };
+  }
+
+  componentDidMount() {
+    const { match: { params: { token } } } = this.props;
+    localStorage.setItem('auth_token', token);
```

Comment:
Since successful login has`auth_token`set in local storage, setting it here would mean that a user can access all the protected routes before the password is actually reset. Kindly consider changing `auth_token` to another variable name.